### ISSUE TITLE
[1.11.2] iChisel Consuming Blocks

### ISSUE THREAD

**Issue by eelusm** [Chisel-Team/Chisel#465]
[https://youtu.be/Bqx9c21K8TU](url)

I noticed a while ago that when I use an iChisel, back then items got consumed/deleted. If you attempt to chisel 32 of whatever block, SHIFT then chisel, the rest of the stacks become 32. Now it becomes 1, whether or not shifting.

A friend mentioned that this was a bug back in 1.10, was wondering if it was ever fixed. 😄 I don't think I need to make an even longer report than this, it's pretty self-explanatory. This was tested on a server, not sure if it's the same in Singleplayer. Haven't tested. ❤️

**Comment by SDUBZ** [comment]
also experiencing this bug is back in chisel -1.11.2-0.0.11.16
Forge:1.11.2-2315
ctm:1.11.2-01.1.6
![2017-06-24_21 32 59](https://user-images.githubusercontent.com/14084377/27508348-10917878-5926-11e7-90ef-890099ce6e70.png)

![2017-06-24_21 33 16](https://user-images.githubusercontent.com/14084377/27508349-14523bfa-5926-11e7-9496-b8f5e92270b1.png)

![2017-06-24_21 33 22](https://user-images.githubusercontent.com/14084377/27508350-17bd1a58-5926-11e7-9011-5c816faede4f.png)

**Comment by tterrag1098** [comment]
I cannot reproduce this in 1.11.2

**Comment by tterrag1098** [comment]
Is this still an issue?

**Comment by kragnoth** [comment]
Happens when you make the chisels have no durability loss for me.  Only thing I can think of that could be causing that behavior, as when I make the chisels take no durability loss, they only leave me one block per chisel.

**Comment by SDUBZ** [comment]
yeah seems to still happen even in newer builds beyong 1.11.2 like @kragnoth said it seems to be when durability is turned off

**Comment by brisingraerowing** [comment]
I noticed this as well in 1.10.2 with the iChisel. I haven't tested in 1.12.2 yet.

**Comment by Mohron** [comment]
Also happens in **1.12.2**. I have durability disabled as well.

Chisel: 1.12-0.1.1.26
Forge: 1.12.2-2587

**Comment by SDUBZ** [comment]
@tterrag1098 have tried also with just chisel itself? having durability disabled

**Comment by mcdodge34** [comment]
I was hoping to save quite a lot of time by using the Chisel all function of the IChisel, turns out its bugged, wish the auto chisel was still a thing, I got a massive scale project where I need around 56800 blocks of a specified chisel type, wonder if a mechanical user or any kind of crafter could do it, but I don't think any would open the interface to select the right version.

**Comment by brisingraerowing** [comment]
I've ended up using Tinkers Complement to get chisels, and a custom crafting recipe for the creative modifier to get them up to 7 modifiers so they can be unbreakable, and add expanders to allow the other in-world chiseling options (take one modifier each).

**Comment by mcdodge34** [comment]
Hum, Tinkers Complement sounds like a nice addition to my adventures, wish I had discovered this one earlier, now for the automated chiseling of a crap ton of blocks, that's gonna be another story I guess

**Comment by blockspiders** [comment]
This is still happening in 1.12.2.

It doesn't happen when in Creative Mode, even though the Chisel is also not damaged there.
It only happens in Survival with damage turned off.

**Comment by WonderDog** [comment]
I have the same issue in 1.12.2 single player survival world

Version Chisel - MC1.12.2-0.2.0.31 in [Flexible Games Builders Pack ](https://minecraft.curseforge.com/projects/flexible-games-builders-pack)

I did update to the latest version of the mod when I first encountered the error.

Durability is disabled and the iron and diamond chisel work. 

The iChisel will only create 1 item for any size stack reducing the stack to 1 item.
